pkgs/development/python-modules/hawkmoth/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchPypi,
+  hatchling,
+  sphinx,
+  libclang,
+}:
+
+buildPythonPackage rec {
+  pname = "hawkmoth";
+  version = "0.19.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-NxjSUg/ZznuAKItzbc/sInZGbQxJLXy0HLyCLxLK8yo=";
+  };
+
+  build-system = [ hatchling ];
+
+  dependencies = [
+    sphinx
+    libclang
+  ];
+
```

Comment:
add pytestCheckHook if the package has tests

Reply:
Added. It took some time to fix tests because they are actually broker in the 0.19.0 release as it fails to correctly locate include paths. Fixed with three patches from master.

Reply:
Got it. Add a comment that they can probably be removed the next bump

Reply:
I haven't fully noted it as TODO but I noted why patch is present. Now it is prefixed with `TODO` and explicitly states that it should be removed with version bump.

Reply:
Updated to 0.20.0 thus patches are removed now.